### PR TITLE
update welcome page

### PR DESCRIPTION
**Pull request by moshekruger**

### CODE REVIEW

**Comment by moshekruger** [comment]
@Himanshusingh22 
Have a look, please

File: docs/index.rst [hunk 1]
```diff
@@ -1,96 +1,119 @@
...
 
-* :ref:`Track <track>`: all metrics and outputs in your data science or machine learning project. It can be model training curves, visualizations, input data, calculated features and so on.
-* :ref:`Organize <organize>`: automatically transform tracked data into a knowledge repository.
-* :ref:`Collaborate <collaborate>`: share, compare and discuss your work across data science project.
+Neptune is a light-weight experiment management tool that helps you keep track of your machine learning experiments.
+
+Use Neptune to log hyperparameters and output metrics from your runs, then visualize and compare results. Automatically transform tracked data into a knowledge repository, then share and discuss your work with colleagues.
+
+- Neptune fits in any workflow, ranging from data exploration and analysis, decision science to machine learning and deep learning.
+- Neptune works with common technologies in the data science domain: Python 2 and 3, `Jupyter Notebooks <https://docs.neptune.ai/notebooks/introduction.html>`_, and `R <https://docs.neptune.ai/integrations/r-support.html>`_, to mention a few.
```
Comment (pitercl) [inline docs/index.rst]: `Python` instead of `Python 2 and 3`

File: docs/index.rst [hunk 2]
```diff
@@ -1,96 +1,119 @@
...
-Neptune's focus: track, organize and collaborate
-------------------------------------------------
-.. _track:
+- **Track** all objects in the data science or machine learning project. It can be model training curves, visualizations, input data, calculated features and so on. The snippet below presents an example of integration with Python code.
 
-Track
-^^^^^
-Track all objects in the data science or machine learning project. It can be model training curves, visualizations, input data, calculated features and so on.
-Snippet below, presents example integration with Python code.
+    .. code-block:: bash
```
Comment (pitercl) [inline docs/index.rst]: Why bash? It's Python.

Comment (pitercl) [inline docs/index.rst]: The entire snippet should be:
```py
import neptune

neptune.init('shared/onboarding', api_token='ANONYMOUS')

neptune.append_tag('minimal-example')
n = 117
for i in range(1, n):
    neptune.send_metric('iteration', i)
    neptune.send_metric('loss', 1/i**0.5)
    neptune.set_property('n_iterations', n)
```

File: docs/index.rst [hunk 3]
```diff
@@ -1,96 +1,119 @@
...
-       neptune.append_tag('minimal-example')
-       n = 117
-       for i in range(1, n):
-           neptune.send_metric('iteration', i)
-           neptune.send_metric('loss', 1/i**0.5)
-       neptune.set_property('n_iterations', n)
 
-``api_token`` belongs to the public user |Neptuner|. So, when started you can see your experiment at the top of |experiments view|.
+    .. note::
+       The `api_token` belongs to the public user Neptuner.
```
Comment (pitercl) [inline docs/index.rst]: I think another sentence is required here. Something along the lines of:
After running the code, your experiment will appear here: https://ui.neptune.ai/shared/onboarding/experiments

File: docs/index.rst [hunk 4]
```diff
@@ -1,96 +1,119 @@
...
+- Sample projects like: a `comparison of binary classification metrics <https://ui.neptune.ai/neptune-ai/binary-classification-metrics/wiki/README-12ff3437-42e3-48c9-af34-957822849559>`_ applied to fraud detection, `research on hyperparameter optimization strategies <https://ui.neptune.ai/jakub-czakon/blog-hpo/wiki/Skopt-forest-51912822-7a61-42ad-87d1-108998739c73>`_, or a `step-by-step experiment tracking tutorial <https://ui.neptune.ai/USERNAME/example-project/wiki/1-Intro-89a74d1e-c71d-4764-912a-63312c3e885c>`_.
+- `YouTube channel <https://www.youtube.com/channel/UCvOJU-ubyUqxGSDRN7xK4Ng>`_: Provides hands-on videos that showcase key Neptune features.
+- `Blog <https://neptune.ai/blog>`_: Provides in-depth articles about best practices and trends in machine learning.
+- `Neptune user community <https://spectrum.chat/neptune-community?tab=posts>`_: Meet other Neptune users and developers and start a discussion.
+- Presentations, talks, podcasts
+- Technical Support: Should you require further support, or have feature requests, reach out at support@neptune.ai or click the chat icon in the bottom right corner of the Neptune UI.
```
Comment (pitercl) [inline docs/index.rst]: There's no such address. I suggest contact@neptune.ai

File: docs/index.rst [hunk 5]
```diff
@@ -1,96 +1,126 @@
...
+- Presentations, talks, podcasts
+- Technical Support: Should you require further support, or have feature requests, reach out at contact@neptune.ai or click the chat icon in the bottom right corner of the Neptune UI.
+
+Spread the Love
+===============
+
+Go ahead and mention us on social media!
+
+- `Twitter <https://twitter.com/neptune_ai>`_: Tweet us. Our handle is @neptune.ai.
+- Product feedback: File an issue in our `GitHub repo <https://github.com/neptune-ai/neptune-client>`_.
```
[the remark this answers is not in the document]
Comment (moshekruger) [reply docs/index.rst]: I had suggested it the first time round and then I saw that what you wrote wasn't very meaningful so left it out.
I've put it back as you wrote it now.
Sorry.

File: docs/index.rst [hunk 6]
```diff
@@ -93,7 +93,8 @@ In addition to this documentation set, check out the following resources:
 - `Blog <https://neptune.ai/blog>`_: Provides in-depth articles about best practices and trends in machine learning.
 - `Neptune user community <https://spectrum.chat/neptune-community?tab=posts>`_: Meet other Neptune users and developers and start a discussion.
 - Presentations, talks, podcasts
-- Technical Support: Should you require further support, or have feature requests, reach out at contact@neptune.ai or click the chat icon in the bottom right corner of the Neptune UI.
+- Product hunt: a review helps other people find our product.
```
Comment (jakubczakon) [inline docs/index.rst]: The link to Product hunt profile is missing here.
https://www.producthunt.com/posts/neptune-ai

Comment (moshekruger) [reply docs/index.rst]: That's what I mean when I said it didn't have enough "meat".
Adding it now.

File: docs/index.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (jakubczakon) [reply docs/index.rst]: Sorry, my bad, must have misunderstood.

**Pull request merged**